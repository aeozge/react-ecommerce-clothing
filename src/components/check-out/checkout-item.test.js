import React from "react";
import CheckOutItem from './checkout-item.component'

const mockProps = {
  cartItem:{
    name: 'women',
    imageUrl: 'www.testImage.com',
    price : 15,
    quantity: 3
  }
}

test('checkout test', () => {
  const wrapper = (<CheckOutItem {...mockProps} />);
  expect(wrapper).toMatchSnapshot()
})
