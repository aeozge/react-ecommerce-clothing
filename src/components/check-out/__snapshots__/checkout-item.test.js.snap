// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checkout test 1`] = `
<CheckOutItem
  cartItem={
    Object {
      "imageUrl": "www.testImage.com",
      "name": "women",
      "price": 15,
      "quantity": 3,
    }
  }
/>
`;
